win32/devdoc/threadpool_win32.md

```diff
@@ -293,12 +293,12 @@ MOCKABLE_FUNCTION(, int, threadpool_schedule_work_item, THANDLE(THREADPOOL), thr
 ### threadpool_destroy_work_item
```

Comment:
>threadpool_destroy_work_item [](http://example.com/codeflow?start=4&length=28)

This should now be threadpool_dispose_work_item

Reply:
We typically name the dispose functions with "dispose" in them to differentiate them somewhat from "destroy" functions - which are meant to be called by the user directly.

Reply:
Done